main.lua

```diff
@@ -45,36 +46,32 @@ love.keypressed = function(pressed_key)
 end
 
 love.draw = function()
-        Camera.set()
+    Camera.set()
 
-        background.draw( love.starLocations)
+    background.draw(love.starLocations)
 
-        map.draw()
-        if State.paused == true then
-            menu.draw()
-        end
+    map.draw()
+
+    if State.paused == true then
+        menu.draw()
+    else
+        HUD.draw()
+    end
 
     Camera.unset()
 end
 
 love.update = function(dt)
     if State.paused == false then
         world:update(dt)
-        love.seconds = love.seconds + dt
+        State.seconds = State.seconds + dt
```

Comment:
Yall got rid of my bad idea

Reply:
Hey it's not a bad idea if all I did was change the name :)